src/components/masters/program/main-content/data/actions.js

```diff
@@ -29,7 +29,7 @@ const clearProgramEnrollmentOverviewEvent = () => ({ type: CLEAR_PROGRAM_COURSE_
 const fetchProgramEnrollmentOverview = options => (
```

Comment:
Let's be more explicit and consistent about the naming of functions. Because the API calls we make are similar enough (i.e., both related to programs), naming things clearly will help clarify which functions do what.

For example, this function could be renamed `fetchProgramCourseEnrollments`. That way, it's more evident what that function is doing and what type of data is returned. As opposed to `fetchProgramEnrollmentOverview`, which I feel like isn't descriptive enough (especially when we have other functions that follow a similar `fetchUserProgramEnrollments` pattern). The renaming here would help make it more obvious which does what.

Similarly, all the function names related to fetching program course enrollments in this file should be renamed to follow the `fetchProgramCourseEnrollments` pattern (e..g, `fetchProgramCourseEnrollmentsRequest`, `fetchProgramCourseEnrollmentsSuccess`, etc.).

Reply:
Sounds good, I believe I renamed everything to reflect this change.